frontend/src/views/TE/Task.vue

```diff
@@ -124,7 +124,9 @@
               </v-row>
             </v-timeline-item>
             <v-timeline-item
-              v-else-if="index === versionsCount - 1 && isCoordinator"
+              v-else-if="
+                index === versionsCount - 1 && $can('update', 'TE/Task')
```

Comment:
Here action is `resolve` (correct my English, the meaning is to carry out resolution).